webservice/src/main/java/pl/edu/pjatk/lnpayments/webservice/auth/config/WebSecurityConfig.java

```diff
@@ -18,11 +18,18 @@
 import org.springframework.web.filter.CorsFilter;
 import pl.edu.pjatk.lnpayments.webservice.auth.filter.AuthTokenFilter;
 
-import static pl.edu.pjatk.lnpayments.webservice.common.Constants.AUTH_PATH;
+import static pl.edu.pjatk.lnpayments.webservice.common.Constants.*;
 
 @Configuration
 @EnableWebSecurity
 class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+    final String[] UNSECURED_PATHS = {
```

Comment:
To pole powinno być private ale tak to git.

Reply:
dżwowcy be like